lib/ddtrace/span_operation.rb

```diff
@@ -15,6 +15,7 @@ module Datadog
   # build a Span. When completed, it yields the Span.
   #
   # rubocop:disable Metrics/ClassLength
```

Comment:
⬆️ The rubocop magic comment is showing up in the generated docs (same for `TraceOperation` and `Tracer`)

Reply:
> The rubocop magic comment is showing up in the generated docs

I feel like this is a common enough thing that YARD should have some support for it out of the box. I noticed this during development, but didn't address each one, because there was no way 🙃.

I couldn't find anything regarding filtering out docstrings, so I'm thinking of proposing something upstream.

In the meantime, I'll write an extension to remove rubocop/magic strings.

Reply:
These should have been removed in the latest commit.